integrations-and-supported-tools/pytorch-lightning/notebooks/Neptune_PyTorch_Lightning.ipynb

```diff
@@ -129,9 +129,9 @@
    "outputs": [],
    "source": [
     "params = {\n",
-    "    \"batch_size\": 32,\n",
+    "    \"batch_size\": 2,\n",
     "    \"lr\": 0.007,\n",
-    "    \"max_epochs\": 15,\n",
+    "    \"max_epochs\": 2,\n",
```

Comment:
it should be 5 :)

Reply:
tests are timing out even with 4 epochs :)
since these are only examples, does it really matter?

Reply:
ok